python/cuml/fil/fil.pyx

```diff
@@ -137,6 +137,12 @@ cdef class TreeliteModel():
         model.set_handle(handle)
         return model
 
+    @staticmethod
+    def from_treelite_model_handle(treelite_handle):
+        cdef ModelHandle handle = <ModelHandle> <size_t> treelite_handle
+        model = TreeliteModel()
+        model.set_handle(handle)
```

Comment:
This appears to give "ownership" of the treelite handle to the new model. So when the new model is deleted, the model pointed to by the handle will be freed. Maybe that's ok, but we should look at the usages and make sure it is, or I think we'll get a double free. Should document it to.

Reply:
The model does not seem to get freed twice, I believe this is because python does not delete the value  it deletes the name. Therefore, as long as there is a variable that points to the value, the value is never freed.